x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -35,27 +104,6 @@ WS
     : [ \r\n\t]+ -> channel(HIDDEN)
     ;
 
-// in 8.14 ` were not allowed
-// this has been relaxed in 8.15 since " is used for quoting
-fragment UNQUOTED_SOURCE_PART
-    : ~[:"=|,[\]/ \t\r\n]
-    | '/' ~[*/] // allow single / but not followed by another / or * which would start a comment -- used in index pattern date spec
-    ;
-
-UNQUOTED_SOURCE
-    : UNQUOTED_SOURCE_PART+
-    ;
-
-//
-// Explain
-//
-mode EXPLAIN_MODE;
-EXPLAIN_OPENING_BRACKET : OPENING_BRACKET -> type(OPENING_BRACKET), pushMode(DEFAULT_MODE);
-EXPLAIN_PIPE : PIPE -> type(PIPE), popMode;
-EXPLAIN_WS : WS -> channel(HIDDEN);
-EXPLAIN_LINE_COMMENT : LINE_COMMENT -> channel(HIDDEN);
-EXPLAIN_MULTILINE_COMMENT : MULTILINE_COMMENT -> channel(HIDDEN);
```

Comment:
I've moved this down since EXPLAIN mode is still in limbo - no need to have it high in the file, before the EXPRESSION mode.
The modes were initially done in alphabetical order however I think it's best to do them in chronological form.